### PR TITLE
Delete packed folder, HotReload packs everything

### PR DESCRIPTION
**Pull request by manavortex**
# Packing improvements

#1723 Can now check a box to remove the `packed` folder after packing:
image

"Hot Reload" will now install all files, not just the .archive

### CODE REVIEW

File: WolvenKit.App/Models/LaunchProfile.cs [hunk 1]
```diff
@@ -16,9 +16,13 @@ public class LaunchProfile
     // install script files
 
     [Category("Install")]
-    [Display(Name = "Clean packed directory completely first")]
+    [Display(Name = "Clean packed directory before build?")]
     public bool CleanAll { get; set; } = true;
 
+    [Category("Install")]
+    [Display(Name = "Clean packed directory after build?")]
```
Comment (rfuzzo) [inline WolvenKit.App/Models/LaunchProfile.cs:23]: what's the point of this exactly? :D why clean after a build?

File: WolvenKit.App/Models/LaunchProfile.cs [hunk 2]
```diff
@@ -16,9 +16,13 @@ public class LaunchProfile
     // install script files
 
     [Category("Install")]
-    [Display(Name = "Clean packed directory completely first")]
+    [Display(Name = "Clean packed directory before build?")]
```
Comment (rfuzzo) [inline WolvenKit.App/Models/LaunchProfile.cs:19]: should that really ever be allowed?

File: WolvenKit.App/Controllers/RED4Controller.cs [hunk 3]
```diff
@@ -269,15 +267,20 @@ private static bool IsCDPRScript(string fileName)
         };
     }
 
-    private static IEnumerable<string> GetScriptFiles(Cp77Project cp77Proj) 
-        => Directory.EnumerateFiles(cp77Proj.ResourceScriptsDirectory, "*.*", SearchOption.AllDirectories).Where(name => IsCDPRScript(name));
+    private static List<string> GetScriptFiles(Cp77Project cp77Proj)
```
Comment (rfuzzo) [inline WolvenKit.App/Controllers/RED4Controller.cs:276]: why the change to List?

File: WolvenKit.App/Models/LaunchProfile.cs (hunk 2 above)
Comment (manavortex) [reply WolvenKit.App/Models/LaunchProfile.cs:19]: It's the current default behaviour :D

File: WolvenKit.App/Models/LaunchProfile.cs (hunk 1 above)
Comment (manavortex) [reply WolvenKit.App/Models/LaunchProfile.cs:23]: because Nola complained that NPV projects clutter up her disk, and tbh she has a point

File: WolvenKit.App/Controllers/RED4Controller.cs (hunk 3 above)
Comment (manavortex) [reply WolvenKit.App/Controllers/RED4Controller.cs:276]: ConcurrentModification warnings when accessing

Comment (rfuzzo) [reply WolvenKit.App/Controllers/RED4Controller.cs:276]: ah right, you made it async 👍

File: WolvenKit.App/Models/LaunchProfile.cs (hunk 2 above)
Comment (rfuzzo) [reply WolvenKit.App/Models/LaunchProfile.cs:19]: eh okay, I guess we could remove that setting

**Review by rfuzzo — APPROVED** [review]

**Review by rfuzzo — CHANGES_REQUESTED** [review]
ah, the regex thing is still unresolved

**Review by rfuzzo — APPROVED** [review]
lgts (looks good to seb)
